packages/ui/src/index.ts

```diff
@@ -7,6 +7,8 @@ export * from './components/button';
 export * from './components/card';
 export * from './components/checkbox';
 export * from './components/dialog';
+export * from './components/data-table';
```

Comment:
we removed this on purpose. are you able to import directly?

Reply:
It wasn't building when i tried to import it directly